lib/appsignal/transaction.ex

```diff
@@ -269,6 +269,30 @@ defmodule Appsignal.Transaction do
     transaction
   end
 
+  @doc """
+  Set namespace of the current transaction. See `set_namespace/1`.
+  """
+  @spec set_namespace(atom()) :: Transaction.t()
```

Comment:
Why limit it to an atom? Namespaces can be strings?

Reply:
Will check the Ruby integration to compare.